### PR TITLE
add xenopsd binary for Rust (use JSON/RPC, server not implemented for…

### PR DESCRIPTION
**Pull request by Wescoeur**
… the moment)

### CODE REVIEW

File: rust/Cargo.toml [hunk 1]
```diff
@@ -1,17 +1,25 @@
 [package]
-name = "xenopsd-rs"
+name = "xenops"
```
Comment (benjamreis) [inline rust/Cargo.toml:2]: `xenops-ng`?

File: rust/daemon/main.rs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    }
+
+    let parsed: VmUnpauseParams = params.parse()?;
+    match xc.lock().unwrap().unpause_domain(parsed.dom_id) {
+      Ok(_) => Ok(Value::String(String::from("success"))),
+      Err(e) => Err(make_error(&e.to_string()))
+    }
+  } } );
+
+  io.add_method("vm.shutdown", enclose! { (xc) move |params: Params| {
```
Comment (benjamreis) [inline rust/daemon/main.rs]: Why enclosing `xc` when you're using `xs` instead?

File: rust/Cargo.toml (hunk 1 above)
Comment (Wescoeur) [reply rust/Cargo.toml:2]: :+1:

Comment (benjamreis) [reply rust/Cargo.toml:2]:
```suggestion
name = "xenops-ng"
```

**Review by benjamreis — APPROVED** [review]

**Pull request merged**